### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeanniefinks**
Multiple editorial edits

### CODE REVIEW

**Review by mgoin — DISMISSED** [review]

**Review by bfineran — APPROVED** [review]

File: examples/amd-azure/README.md [hunk 1]
```diff
@@ -39,58 +42,59 @@ The throughput value reported comes from measuring the number of finished infere
...
 
-Running with `multi_process_benchmark.py -h` or `--help` provides usage options:
+Executing `multi_process_benchmark.py -h` or `--help` provides the following usage options:
 
 ```
 usage: python multi_process_benchmark.py [-h] [-b BATCH_SIZE] [-nstreams NUM_STREAMS] [-shapes INPUT_SHAPES] [-t TIME] [-w WARMUP_TIME]
                                   [-pin {none,core,numa}] [-q]
                                   model_path topology_file
 
-Benchmark ONNX models in the DeepSparse Engine
```
Comment (tlrmchlsmth) [inline examples/amd-azure/README.md:71]: Everything from here down should be consistent with the parse_args function in multi_process_benchmark.py

**Review by tlrmchlsmth — CHANGES_REQUESTED** [review]

File: examples/amd-azure/README.md (hunk 1 above)
Comment (jeanniefinks) [reply examples/amd-azure/README.md:71]: I can apply these edits to that file @tlrmchlsmth

Comment (tlrmchlsmth) [reply examples/amd-azure/README.md:71]: Let me know if you need any help

Comment (jeanniefinks) [reply examples/amd-azure/README.md:71]: Made changes and will ask for a re-review. parse_args for -h and -help must come default from elsewhere and I didn't update the descriptive language there; honestly, not a showstopper.

**Review by tlrmchlsmth — APPROVED** [review]
lgtm, thanks!

File: examples/amd-azure/README.md [hunk 2]
```diff
@@ -39,58 +42,59 @@ The throughput value reported comes from measuring the number of finished infere
...
 
-In most cases, good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model.
+In most cases, good performance will be found in the default options so it can be as simple as running the command with a [Neural Magic SparseZoo](https://sparsezoo.neuralmagic.com) model stub or your local ONNX model.
 
 ```bash
 python multi_process_benchmark.py <path/to/model> <path/to/topology/file>
 ```
 
-Running with `multi_process_benchmark.py -h` or `--help` provides usage options:
+Executing `multi_process_benchmark.py -h` or `--help` provides the following usage options:
```
Comment (bnellnm) [inline examples/amd-azure/README.md:68]: "or `help`" -> "or with `--help`"?

**Review by tlrmchlsmth — APPROVED** [review]

**Review by bnellnm — DISMISSED** [review]
LGTM.  just a small question about the wording for getting help with the benchmark script.  I think it's fine either way tho.

File: examples/amd-azure/README.md (hunk 2 above)
Comment (jeanniefinks) [reply examples/amd-azure/README.md]: sure, fair enough.

**Review by tlrmchlsmth — APPROVED** [review]

**Review by bnellnm — APPROVED** [review]

**Pull request merged**